doc/sphinxext/ipython_sphinxext/ipython_directive.py

```diff
@@ -498,6 +494,16 @@ def process_input(self, data, input_prompt, lineno):
                 sys.stdout.write(s)
                 sys.stdout.write('<<<' + ('-' * 73) + '\n')
 
+        # if :okexcept: has been specified, display shorter traceback
+        if is_okexcept and "Traceback" in output:
```

Comment:
@kawochen Thanks for the idea. That would indeed be a solution to also get the SyntaxError working with this.

But, it does not really explain why this just aborts on travis, because I left out the SyntaxError with the latest commit ..